docs/plugins/repository-s3.asciidoc

```diff
@@ -210,6 +210,13 @@ The following settings are supported:
 
     Makes repository read-only. coming[2.1.0]  Defaults to `false`.
 
+`storage_class`::
+
+    Sets the S3 storage class type for the backup files. Values may be
+    `STANDARD`, `REDUCED_REDUNDANCY`, `GLACIER`, `STANDARD_IA`.
```

Comment:
a common practice at elasticsearch is to have lower case options.

I think we should document the lowercase version and do a toUpper() in `S3Repository` class. So whatever the user enter (lowercase or uppercase), it will work.